css/style.css

```
@@ -16,4 +16,169 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    height: 40px;
+    margin: 30px;
+    padding: 0 40px;
+}
+.logo {
+    width: 40px;
+    height: 30px;
+}
+
+.menu-burger {
+    visibility: hidden;
+}
+
+ul {
+    list-style-type: none;
+}
+
+.nav-wrapper {
+    display: flex;
+    flex-direction :row;
+    flex-wrap : nowrap;
+    justify-content:flex-end;
+    align-items:flex-start;
+    align-content:space-between;
```

```suggestion
    flex-wrap : nowrap;
    justify-content:flex-end;
    align-items:flex-start;
    align-content:space-between;
```
Do you need these? What will happen if we don't have them?